apps/L1E/instagram/instagram-backend/src/models/User.ts

```
@@ -0,0 +1,29 @@
+export interface IUser extends Document {
+  email: string;
+  password: string;
+  username: string;
+  bio?: string;
+  phone?: string;
+  gender?: string;
+  profilePhoto?: string;
+  followers: mongoose.Types.ObjectId[];
+  following: mongoose.Types.ObjectId[];
+  createdAt: Date;
+  comparePassword(candidatePassword: string): Promise<boolean>;
```

ESLint

[error] 'candidatePassword' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)